69.md

```diff
@@ -0,0 +1,205 @@
+
+Some of the content types involve "in the eye of the beholder" judgement calls. Relay owners and Nostr-based communities will use and/or ignore reports only as the reports meet their particular standards. There won't be a single standard on any of these items, but rather thousands of standards.
+
+To reiterate - there is no way to censor Nostr. This isn't about censorship. It's about facilitating the content moderation necessary to be compliant with local laws and community standards that define the community of users for a particular relay or client app.
+
+
+
+Reportable Types:
+
+These types expand and standardize the "report types" defined in NIP-56. Existing NIP-56 report types (shown below as "sometype | ...") should be supported by clients for backwards compatibility, but should not otherwise be used in future development.
+
+Format: AB-cde
+ - where 'AB' is a mandatory 2 character, uppercase code of a major content type category
+ - and 'cde' is an optional 3 character, lowercase code representing a sub-category
+
+
+profanity | CL - Coarse Language / Profanity
+IH - Intolerance & Hate (does not include intolerance of intolerance)
+illegal | IL - Illegal (post/profile is illegal or advocates illegal activity)
+IL-cop - Copyright violation, piracy, intellectual property theft
+IL-csa - Child sexual abuse and/or trafficking
+IL-drg - Drug-related crime
+IL-frd - Fraud & Scams
+IL-har - Harassment / stalking / doxxing
+IL-hkr - Prostitution
+IL-idt - Identity theft / phishing
+IL-mal - Malware / viruses / ransomware
+impersonation | IM - Impersonation (consider also IL-idp)
+MI-mny - Promotion of misinformation that is likely to cause financial ruin (consider also IL-frd)
+MI-hth - Promotion of misinformation that is likely to cause serious bodily harm or death
+nudity | NS - Nudity & Sex
+NS-nud - Casual nudity
+NS-ero - Erotica
+NS-sex - Sex
+PN - Pornography
+PN-het - Heterosexual porn
+PN-gay - Gay male porn
+PN-les - Lesbian porn
+PN-bis - Bisexual porn
+PN-trn - Transexual porn
+PN-fnb - Gender-fluid / non-binary porn
+spam | SP - Spam
+SP-mod - Moderation report spam
+VI - Violence (actual or advocated)
+VI-hum - Violence towards a human being
+VI-ani - Violence towards a sentient animal
+
+
+
+Contexts:
+
+Contexts are situations where otherwise objectionable content may be allowed.
+
+ED - Educational
+FA - Fine Art
+FF - Fantasy / Fiction
+MS - Medical / Scientific
+ND - News & Documentaries
+PP - Political Protest
+
+
+
+Use with NIP-56 3rd party reports:
+
+To use with NIP-56 simply replace the original 5 defined report types with the 2 or 6 character codes above.
+
+The one change to NIP-56 (or clarification) is that the value passed may be a comma delimited list to support specifying multiple report types or contexts.
+
+
+
+Use with NIP-36 self-reporting:
+
```

Comment:
```
tag: content-warning
options:
 - [reason]: may be a null or blank
 - [codes]: mandatory
```
The [reason] option was defined in NIP-36. It may be null or a blank string.